design-proposals/README.md

```diff
@@ -13,12 +13,13 @@ Here is the process for submitting a KLIP and getting it approved.
 
 # KsqL Improvement Proposals (aka KLIPs)
 
-Next KLIP number: 2.
+Next KLIP number: 5.
```

Comment:
I will update the numbers if this is merged before the currently open KLIPS: (#2553, #2679)